app/preload.js

```diff
@@ -16,15 +16,24 @@ function navigate(url) {
 
 ipcRenderer.on('isPlaying', (event) => {
   const isPlaying = !!document.querySelector('.playing')
-  event.sender.send('isPlaying', isPlaying)
+  const icon = document.getElementsByClassName('playbackSoundBadge__avatar sc-media-image')[0].children[0].children[0].getAttribute('style')
+  var urlStart = icon.indexOf('url("') + 5;
+  var urlEnd = icon.indexOf('");') - urlStart;
+  var iconURL = icon.substr(urlStart, urlEnd)
```

Comment:
Extracting the url from the stylesheet might fail in many ways. There might not be a background image at all, the syntax might be slightly different, there might be (single or double) quotes around the url or not, `URL()` might be uppercase, etc.

To be on the safe side I'd suggest using a RegEx to parse the url from the stylesheet and add some error handling (do not fail if we could not actually find a url).

Something like this:
```js
const match = icon.style.backgroundImage.match(/(?:url\s*\(\s*['"]?)(.*?)(?:['"]?\s*\))/i)
const iconURL = match && match[1]
```